fuzz/build.rs

```diff
@@ -0,0 +1,9 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+fn main() {
```

Comment:
Do you list build.rs in Cargo.toml?  Should you?

Reply:
Hm. It doesn't look like it's needed, in the sense that it takes effect. The [documentation](https://doc.rust-lang.org/cargo/reference/manifest.html#the-build-field) seems to imply that `build.rs` is picked up by default, even if not specified. (We can even remove the `build` lines from `neqo-common` and `neqo-crypto` and things still build fine.)